packages/fulfillment/src/models/shipping-option.ts

```diff
@@ -30,46 +30,44 @@ import ShippingProfile from "./shipping-profile"
 
 type ShippingOptionOptionalProps = DAL.SoftDeletableEntityDateColumns
 
-const deletedAtIndexName = "IDX_shipping_option_deleted_at"
-const deletedAtIndexStatement = createPsqlIndexStatementHelper({
-  name: deletedAtIndexName,
+const DeletedAtIndex = createPsqlIndexStatementHelper({
   tableName: "shipping_option",
   columns: "deleted_at",
   where: "deleted_at IS NOT NULL",
-}).expression
+})
 
-const serviceZoneIdIndexName = "IDX_shipping_option_service_zone_id"
-const serviceZoneIdIndexStatement = createPsqlIndexStatementHelper({
-  name: serviceZoneIdIndexName,
+const ServiceZoneIdIndex = createPsqlIndexStatementHelper({
   tableName: "shipping_option",
   columns: "service_zone_id",
   where: "deleted_at IS NULL",
-}).expression
+})
 
-const shippingProfileIdIndexName = "IDX_shipping_option_shipping_profile_id"
-const shippingProfileIdIndexStatement = createPsqlIndexStatementHelper({
-  name: shippingProfileIdIndexName,
+const ShippingProfileIdIndex = createPsqlIndexStatementHelper({
   tableName: "shipping_option",
   columns: "shipping_profile_id",
   where: "deleted_at IS NULL",
-}).expression
+})
 
-const serviceProviderIdIndexName = "IDX_shipping_option_service_provider_id"
-const serviceProviderIdIndexStatement = createPsqlIndexStatementHelper({
-  name: serviceProviderIdIndexName,
+const ServiceProviderIdIndex = createPsqlIndexStatementHelper({
   tableName: "shipping_option",
   columns: "service_provider_id",
   where: "deleted_at IS NULL",
-}).expression
+})
 
-const shippingOptionTypeIdIndexName =
-  "IDX_shipping_option_shipping_option_type_id"
-const shippingOptionTypeIdIndexStatement = createPsqlIndexStatementHelper({
-  name: shippingOptionTypeIdIndexName,
+const ShippingOptionTypeIdIndex = createPsqlIndexStatementHelper({
   tableName: "shipping_option",
   columns: "shipping_option_type_id",
   where: "deleted_at IS NULL",
-}).expression
+})
+
+const nameIndexName = "IDX_shipping_option_name_unique"
```

Comment:
And it shouldn't even be unique for each service zone for example you could have 2 express shipping - one for b2b and one for b2c; you'd use the shipping option rules to make them available to each.

Reply:
I ll change that, my thinking was that they would be name differently in those cases in order to prevent miss understanding if they have the exact same name as they serve different purposes. I don't have a strong opinion on that and will rm the uniqueness